build/ProjectReunion-BuildFoundation.yml

```diff
@@ -13,8 +13,7 @@ parameters:
 
 jobs:
 - job: CredScan
-  pool:
-    vmImage: 'windows-2019'
+  pool: $(ProjectReunionBuildPool)
```

Comment:
>ProjectReunionBuildPool [](http://example.com/codeflow?start=10&length=23)

do we need to have a default value in yml?

Reply:
I don't think we need to have one.